spec/core-spec.md

```diff
@@ -66,7 +66,7 @@ This mapping will contain the element `symbol` (string), and the `atomic_mass` (
 The element table may also specify atomic numbers `N` and `Z` as integers, allowing for multiple isotopes of species.
 If `N` is provided the model should be considered as a model of a particular isotope, and not a general case for a species.
```

Comment:
The terminology of `N` could be a bit confusing. Is it the neutron number, or the mass number? Maybe best to add an optional field `mass_number` or `A` instead.

Reply:
`N` is intended to be the neutron number, but I see the space for confusion!

Reply:
I don't know what the best way is to handle isotopes. Maybe just a different atomic mass instead of `N` or `A`? We also need to submit the fraction for each isotope. Not sure if `N` is actually needed for any calculation.